include/kodlab_mjbots_sdk/soft_start.h

```diff
@@ -16,7 +16,7 @@ class SoftStart {
  private:
   float slope_ = 0;        /// Slope of the ramp
   float max_torque_ = 100; /// Max torque allowed in Nm
-  int duration_ = 0;       /// Duration of soft Start in steps
+  float duration_ms_ = 0;   /// Duration of soft Start in steps
```

Comment:
fix comment

Reply:
Fixed in 21511f7a0d652fe35a9fc578d4b151ac0b83c9d4